src/libs/SidebarUtils.ts

```diff
@@ -173,8 +174,14 @@ function getOrderedReportIDs(
     const nonArchivedReports: Report[] = [];
     const archivedReports: Report[] = [];
 
+    let workspaceReportsToDisplay: Report[] = reportsToDisplay;
+
+    if (currentPolicyID) {
+        workspaceReportsToDisplay = workspaceReportsToDisplay.filter((report) => report.policyID === currentPolicyID);
```

Comment:
Why do we need a new variable for this, I think we can keep it same `reportsToDisplay` here